XSLT-to-SVG-1/Love_XSLT-SVG/Love_SVGEX2.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
@dal5842 I think you meant to put this one in the other folder. Check where your output SVGs went!